SyntaxWarning: "is not" with a literal. Did you mean "!="?

Patch from upstream:
https://github.com/giampaolo/psutil/commit/3d12e67a9e4af3328660d4b3a81966555d7c98bf

--- psutil-6.1.0/psutil/tests/test_system.py
+++ psutil-6.1.0/psutil/tests/test_system.py
@@ -515,7 +515,6 @@ class TestCpuAPIs(PsutilTestCase):
         try:
             assert isinstance(percent, float)
             assert percent >= 0.0
-            assert percent is not -0.0
             assert percent <= 100.0 * psutil.cpu_count()
         except AssertionError as err:
             raise AssertionError(
